pkg/util/cert/cert.go

```diff
@@ -0,0 +1,27 @@
+package cert
```

Comment:
nit: might be good in a follow-up to put these in either `pkg/util/pem` or `/pkg/util/pki` - these feel like a good fit there.

Reply:
Will take this up in a different PR, thank you.